app/views/content/life-as-a-teacher/age-groups-and-specialisms/secondary/_article.html.erb

```diff
@@ -83,6 +83,9 @@
    <h3>Apply for a paid internship</h3>
    <p>If you're currently doing an undergraduate or master's degree and are interested in chemistry, computing, languages, maths or physics, you could apply for a <a href="/train-to-be-a-teacher/teaching-internships">paid teaching internship</a>.</p>
    <p>The 3-week programme could help you to understand what it's really like in the classroom and get a feel for school life.</p></div>
+   <h3>Find secondary teacher training courses<h3>
+    <p>Explore the different secondary courses available.</p>
```

Comment:
Again, this is a bit sparse and repetitive - but I'm not sure I can think of more to say!